.babelrc

```diff
@@ -1,6 +1,5 @@
 {
   "presets": [
-    "@babel/preset-env",
```

Comment:
why is this not needed?

Reply:
I'd ask you why *is it* needed? 😃

Reply:
I guess you already know why I added it 3 years ago so can you be more specific?

Reply:
The question is not why it was added, because it's to support older browser. The question is what browsers do you intend to support today? Because I don’t think you need this transform for those browsers.

Reply:
> without actually having to care about it

That's a partially incorrect assumption, the syntax may be accepted but any for any API you'd still have to _care about it._

> To be honest, not sure why I accepted the runtime transformer PR you contributed first when clearly the problem was not the browserlist.

You keep missing the point, please allow me to help with this matter. The reason why it was necessary was clearly outlined in my original PR.

---

To reiterate and answer both questions:

- The built code refers to a `regeneratorRuntime` global. Please open [the dist/index.min.js](https://www.unpkg.com/browse/react-select-virtualized@5.1.2/dist/index.min.js) and look for it.
- Your babel introduces this global and it should be your build’s responsibility to include it. It's counterintuitive to say "not care about browser support" if the code refers to a global that does not exist in any browser.
- The existence of `regeneratorRuntime` means that the browserlist does not work as you expect it to, because preset-env is still turning `async/await` code into `regenerator` code.

---

To resolve, if you do want to preserve preset-env (which is fine), the browserlist should be updated to match exactly what you're looking for. My suggestion is:

- last 2 chrome version
- last 2 firefox version
- last 2 safari versions

I can restore that